src/components/Common/FilePreviewDialog.tsx

```diff
@@ -428,115 +431,133 @@ const FilePreviewDialog = (props: FilePreviewProps) => {
             </div>
-            <div className="flex items-center justify-center">
-              <div className="mt-2 grid grid-cols-5 max-md:grid-cols-6 gap-4">
+            <div className="mt-4 flex flex-col items-center space-y-2">
+              <div className="flex justify-center space-x-2">
                 {file_state.isImage && (
                   <>
-                    {[
-                      [
-                        t("zoom_in"),
-                        "l-search-plus",
-                        handleZoomIn,
-                        file_state.zoom === zoom_values.length,
-                      ],
-                      [
-                        `${25 * file_state.zoom}%`,
-                        false,
-                        () => {
-                          setFileState({ ...file_state, zoom: 4 });
-                        },
-                        false,
-                      ],
-                      [
-                        t("zoom_out"),
-                        "l-search-minus",
-                        handleZoomOut,
-                        file_state.zoom === 1,
-                      ],
-                      [
-                        t("rotate_left"),
-                        "l-corner-up-left",
-                        () => handleRotate(-90),
-                        false,
-                      ],
-                      [
-                        t("rotate_right"),
-                        "l-corner-up-right",
-                        () => handleRotate(90),
-                        false,
-                      ],
-                    ].map((button, index) => (
-                      <Button
-                        variant="ghost"
-                        key={index}
-                        onClick={button[2] as () => void}
-                        className={cn(
-                          "z-50 rounded bg-white/60 text-black backdrop-blur-sm transition hover:bg-white/70",
-                          index > 2 ? "max-md:col-span-3" : "max-md:col-span-2",
-                        )}
-                        disabled={button[3] as boolean}
-                      >
-                        <div>
-                          {button[1] && (
-                            <CareIcon
-                              icon={button[1] as IconName}
-                              className="text-lg"
-                            />
-                          )}
-                          <div>{button[0] as string}</div>
-                        </div>
-                      </Button>
-                    ))}
+                    <Button
+                      variant="ghost"
+                      onClick={handleZoomOut}
+                      disabled={file_state.zoom === 1}
+                      className="z-50 rounded bg-white/60 px-4 py-2 text-black backdrop-blur-sm transition hover:bg-white/70"
+                    >
+                      <CareIcon
+                        icon={"l-search-minus" as IconName}
```

Comment:
why? 🤔 

if `l-search-minus` exists in the icon library, it'll not have any type issues!

```suggestion
                        icon="l-search-minus"
```